src/main/java/org/broadinstitute/consent/http/service/EmailNotifierService.java

```diff
@@ -238,6 +238,26 @@ public void sendDataCustodianApprovalMessage(String toAddress,
         }
     }
 
+    public void sendDatasetApprovedMessage(String toAddress,
+                                           String dataSubmitterName,
+                                           String dacName,
+                                           String datasetName) throws Exception {
+        if (isServiceActive) {
+            Writer template = templateHelper.getDatasetApprovedTemplate(dataSubmitterName, datasetName, dacName);
+            sendGridAPI.sendDatasetApprovedMessage(toAddress, template);
+        }
+    }
```

Comment:
Just thinking out loud here. Similar case in the next method too. The if is unnecessary here since `sendGridAPI` wraps the sending in the same if condition - the only benefit here is to avoid unnecessary `Writer` construction. In my PR, I actually want to generate the writer in ALL cases so we can capture what would have been sent. I essentially ripped all the if's in this class out for that reason. No need to change this PR, just wanted to clarify what was happening here.